pkg/api/handlers/threads.go

```diff
@@ -66,6 +66,8 @@ func convertThread(thread v1.Thread) types.Thread {
 		Project:         thread.Spec.Project,
 		Env:             env,
 		Ready:           thread.Status.Created,
+		Model:           thread.Spec.Manifest.Model,
+		ModelProvider:   thread.Spec.Manifest.ModelProvider,
```

Comment:
TODO: probably unneeded since it is already in the manifest

Reply:
Yeah, I say remove these fields.